token-swap/program/src/state.rs

```diff
@@ -7,6 +7,8 @@ use solana_sdk::pubkey::Pubkey;
 #[repr(C)]
 #[derive(Clone, Copy, Debug, Default, PartialEq)]
 pub struct SwapInfo {
+    /// Nonce used in construction of program_address
```

Comment:
Can you make this comment a little more explicit? What is the program_address used for?

Reply:
Good point, I'll add some more info